microsoft/testsuites/power/power.py

```diff
@@ -0,0 +1,514 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re, time
+
+from assertpy import assert_that
+
+from lisa import (
+    Node,
+    TestSuite,
+    TestSuiteMetadata,
+    TestCaseMetadata,
+    Environment
+)
+from lisa.base_tools.cat import Cat
+from lisa.operating_system import SLES, CentOs, Redhat
+from lisa.testsuite import TestCaseMetadata
+from lisa.tools.echo import Echo
+from lisa.util import SkippedException
+from lisa.util.logger import Logger
+
+def check_vm_feature_support_status(min_support_version: str, node: Node) -> bool:
+    if not min_support_version:
+        return False
+    
+    kernel_array = node.execute(
```

Comment:
use `node.information.version` It contains the distro version, and the uname tool can return a structured `kernel_version`. The versions can be compared like `> "3.10.0"`.

Reply:
If I understand correctly, these are 2 approaches to get the kernel version? I just need to pick either way?

Reply:
One is the distro version like "7.9" of RedHat, another one is the kernel version like "3.10.0"